src/components/LoginPage.js

```diff
@@ -0,0 +1,66 @@
+import LoginBg from './LoginBg';
+import HeaderNav from './HeaderNav';
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+const LoginPage = () => {
+    
+    console.log('LoginPage has rendered.');
+
+    const [login, setLogin] = useState(false);
+
+    const handleClick = () => {
+        setLogin(!login);
+
+        // change url based on login state
+        if (login === false) {
+            navigate('/home-page');
+        } else if (login === true) {
+            navigate('/');
+        }
+    }
+
+    const navigate = useNavigate();
+
+    return (
+        <>
+            <header className="headerNav">
+                <div className="wrapper">
+                    <h1>NTS</h1>
+                    
+                    <div className="headerRightContainer">
+                        {
+                            login
+                            ? <HeaderNav />
+                            : null
+                        }
+
+                        {/* change button text depending on login state */}
+                        <button className="loginBtn" onClick={handleClick}>{
+                            login
```

Comment:
I have no doubt you'll be able to add users/auth to your login component! The rest of your code is so well structured, thorough and well thought-out!